### ISSUE TITLE
Error and available updates are not shown

### ISSUE THREAD

**Issue by JackLeEmmerdeur** [dhruv8sh/arch-update-checker#5]
I'm using fish shell.
Paru and Yay are installed.

The following error happens both on installing via source or widget download dialog:

* The circular loading bar runs forever and no updates are shown in the widget-dialog.
* A red rectangle with the following error is shown in the widget dialog:

```
Error code: 0
Error:Fehler: Ungültige Kennung : Name darf nicht leer sein
sed: -e Ausdruck #1, Zeichen 0: Kein vorheriger regulärer Ausdruck
```

which translates to:
```
Error code: 0
Error:Error: Invalid identifier: Name can not be empty
sed: -e Expression #1, Char 0: No previous regular expression
```

**Comment by dhruv8sh** [comment]
Wierd. Something similar was reported in #4. Seems to be a flatpak specific problem.
It needs some testing. I will look into this soon.
Until then you can use the applet with flatpak disabled.

**Comment by JackLeEmmerdeur** [comment]
Nice. That did it!
I haven't used flatpak under arch AFAIK, although the flatpak-package from the extra repository is installed but none of the auxiliary flatpak packages.
Thanks for your work.
